code/datums/jobs/job/sons_of_mars_ert.dm

```diff
@@ -342,6 +343,8 @@
 
 /datum/outfit/job/som/ert/veteran/breacher_vet/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	. = ..()
+	H.add_traits(list(TRAIT_AXE_EXPERT, TRAIT_SWORD_EXPERT), TRAIT_GENERIC)
```

Comment:
shouldnt all these traits be trait inate?
And shouldnt they be on the job?

Reply:
I'll update it, whats the diff tho? Also they are on the job?

Reply:
No they're on the outfit
last i checked we apply skills based on job datum

Reply:
Oh whoops, updated